e2e-tests/commands/insurance/assert-currency-form-fields.js

```diff
@@ -85,24 +85,24 @@ const assertCurrencyFormFields = ({
   alternativeCurrencyInput: () => {
     cy.clickAlternativeCurrencyRadioOption();
 
-    checkAutocompleteInput.hasWorkingClientSideJS(countryInput.field(alternativeCurrencyFieldId));
-    checkAutocompleteInput.rendersInput(countryInput.field(alternativeCurrencyFieldId));
+    checkAutocompleteInput.hasWorkingClientSideJS(autoCompleteField(alternativeCurrencyFieldId));
+    checkAutocompleteInput.rendersInput(autoCompleteField(alternativeCurrencyFieldId));
   },
   doesNotRenderSupportedCurrencies: () => {
     cy.clickAlternativeCurrencyRadioOption();
 
-    checkAutocompleteInput.rendersNoResultsMessage(countryInput.field(alternativeCurrencyFieldId), 'not a currency');
+    checkAutocompleteInput.rendersNoResultsMessage(autoCompleteField(alternativeCurrencyFieldId), 'not a currency');
     // should not render radio values in alternate currency input
-    checkAutocompleteInput.rendersNoResultsMessage(countryInput.field(alternativeCurrencyFieldId), GBP.isoCode);
-    checkAutocompleteInput.rendersNoResultsMessage(countryInput.field(alternativeCurrencyFieldId), USD.isoCode);
-    checkAutocompleteInput.rendersNoResultsMessage(countryInput.field(alternativeCurrencyFieldId), JPY.isoCode);
-    checkAutocompleteInput.rendersNoResultsMessage(countryInput.field(alternativeCurrencyFieldId), EUR.isoCode);
+    checkAutocompleteInput.rendersNoResultsMessage(autoCompleteField(alternativeCurrencyFieldId), GBP.isoCode);
```

Comment:
is it worth doing `const autoCompleteField= autoCompleteField(alternativeCurrencyFieldId)` so less complicated line?

Reply:
Good shout, i've made some updates so it's use a `field` variable.

Also noticed that there is some dupication for these assertions and another assertion file, made a note to address this later.